### PR TITLE
feat: Add initiative application section

### PR DESCRIPTION
**Pull request by usame-algan**
## What it solves

Adds a section for initiative application to the careers page

## Screenshot
Screenshot 2022-04-27 at 13 54 52

### CODE REVIEW

File: src/components/careers/CTA/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  position: relative;
+  margin-left: 32px;
+  top: 12px;
+`
+
+const LinkContainer = styled.div`
+  position: relative;
+`
+
+const CallToAction = () => {
```
Comment (iamacook) [inline src/components/careers/CTA/index.tsx:54]: I think we should rename this.

**Review by iamacook — APPROVED** [review]
Looks great!

File: src/components/careers/CTA/index.tsx (hunk 1 above)
Comment (usame-algan) [reply src/components/careers/CTA/index.tsx:54]: Any suggestions?

Comment (iamacook) [reply src/components/careers/CTA/index.tsx:54]: `InitiativeApplication`?

Comment (usame-algan) [reply src/components/careers/CTA/index.tsx]: done

**Pull request merged**